client/components/editor/ActivityContent/index.vue

```diff
@@ -70,11 +71,11 @@ export default {
     ...mapState({ user: state => state.auth.user }),
     activityId: vm => vm.activity.id,
     processedElements() {
-      const { elements, seen, activityId } = this;
+      const { elements, seen, activity, activityId } = this;
       return transform(elements, (acc, it) => {
         const comments = this.getComments({ activityId, contentElementId: it.id });
-        const lastSeen = seen.contentElement[it.uid] || 0;
-        acc[it.uid] = { ...it, comments, lastSeen };
+        const allSeen = [seen.contentElement[it.uid], seen.activity[activity.uid]];
```

Comment:
no need to prefix with all (seen === allSeen). In this case, it is not even allSeen so I would rather do max on this line and have the old name back. Thoughts? 🤔

Reply:
I agree. We can also rename `allSeen` to maybe `seenUids`, I think it's descriptive enough. Gather all seen `uids`, then assign `max` timestamp to lastSeen?